tests/ArgentPonyWarcraftClassicClient.Tests/GameDataApi/AuctionHouseApiTests.cs

```diff
@@ -3,13 +3,62 @@
 public class AuctionHouseApiTests
 {
     [Fact]
-    public async Task GetAuctionsAsync_Gets_Auctions()
+    public async Task GetAuctionHousesAsync_Gets_AuctionHouseIndex_ForBCC()
     {
+        int realmId = 4372;
+        string @namespace = "dynamic-classic-us";
+
+        IAuctionHouseApi warcraftClient = ClientFactory.BuildMockClient(
+            requestUri: $"https://us.api.blizzard.com/data/wow/connected-realm/{realmId}/auctions/index?namespace={@namespace}&locale=en_US",
+            responseContent: Resources.AuctionIndexResponse);
+
+        RequestResult<AuctionHouseIndex> result = await warcraftClient.GetAuctionHousesAsync(realmId, @namespace);
+        Assert.NotNull(result.Value);
+    }
+
+    [Fact]
+    public async Task GetAuctionHousesAsync_Gets_AuctionHouseIndex_ForEra()
+    {
+        int realmId = 5121;
+        string @namespace = "dynamic-classic1x-us";
+
+        IAuctionHouseApi warcraftClient = ClientFactory.BuildMockClient(
+            requestUri: $"https://us.api.blizzard.com/data/wow/connected-realm/{realmId}/auctions/index?namespace={@namespace}&locale=en_US",
+            responseContent: Resources.AuctionIndexResponse);
+
+        RequestResult<AuctionHouseIndex> result = await warcraftClient.GetAuctionHousesAsync(realmId, @namespace);
+        Assert.NotNull(result.Value);
+    }
+
+    [Fact]
+    public async Task GetAuctionsAsync_Gets_Auctions_ForBCC()
+    {
+        int realmId = 4386; 
+        int auctionHouseId = 2;
+        string @namespace = "dynamic-classic-us";
+
+        IAuctionHouseApi warcraftClient = ClientFactory.BuildMockClient(
+            requestUri: $"https://us.api.blizzard.com/data/wow/connected-realm/{realmId}/auctions/{auctionHouseId}?namespace={@namespace}&locale=en_US",
+            responseContent: Resources.AuctionsResponse);
```

Comment:
Should we have different response resources for Classic vs. BCC?  Having separate tests for the integration tests makes good sense.  For the unit tests, it seems like it would only make sense if we have different response content.

Reply:
I also think that different response resources for Classic and BCC are better. I will create them with the next pull request.